### PR TITLE
daniele77-cli: add recipe

### PR DESCRIPTION
**Pull request by toge**
### Summary
Changes to recipe:  **daniele77-cli/2.1**

#### Motivation
daniele77/cli is a CLI interface library.

#### Details
https://github.com/daniele77/cli

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

File: recipes/daniele77-cli/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        copy(
+            self,
+            "*.h",
+            os.path.join(self.source_folder, "include"),
+            os.path.join(self.package_folder, "include"),
+        )
+
+    def package_info(self):
+        self.cpp_info.bindirs = []
+        self.cpp_info.libdirs = []
```
Comment (perseoGI) [inline recipes/daniele77-cli/all/conanfile.py:67]: https://github.com/daniele77/cli/blob/v2.1.0/CMakeLists.txt#L94

File: recipes/daniele77-cli/all/conandata.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "2.1":
```
Comment (perseoGI) [inline recipes/daniele77-cli/all/conandata.yml]: https://github.com/daniele77/cli/releases/tag/v2.1.0

**Review by perseoGI — APPROVED** [review]

File: recipes/daniele77-cli/all/conandata.yml (hunk 2 above)
Comment (toge) [reply recipes/daniele77-cli/all/conandata.yml]: @perseoGI 
Thanks for fixing my mistake so quickly!

**Comment by perseoGI** [comment]
> Library already exists in CCI https://github.com/conan-io/conan-center-index/blob/master/recipes/cli/all/conandata.yml

Oh hehehe that is true! Thank you very much for your good catch! 

@toge I'll close this PR as it exists! 
Thank you as always.

**Comment by toge** [comment]
@perseoGI 
I'm sorry. 
I missed it.

@maksim-petukhov 
Thank you for your pointed out!

**Pull request closed without merging**